src/render.js

```diff
@@ -10,13 +10,27 @@ import options from './options';
  * @param {import('./internal').PreactElement} parentDom The DOM element to
```

Comment:
Hey, nice work one little nitty remark. Could you add the JSDoc types for `replaceNode`

Reply:
Ahh right. Done. I used the comment from `8.x`